### PR TITLE
RDoc-2590 Cloud: Cloud -> Pricing, Payment and Billing tab - add PayNow feature

### PR DESCRIPTION
**Pull request by matz985**
[RDoc-2590](https://issues.hibernatingrhinos.com/issue/RDoc-2590)

image

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 1]
```diff
@@ -162,8 +163,47 @@ cost, and the expected charge at the end of this month.
...
+If you have resolved the issue associated with the charge failure (e.g., updated payment information), you can manually initiate the payment by clicking the `Pay Now` button.
+Upon clicking the `Pay Now` button, the system will process the payment for all outstanding amounts up to the end of the previous billing cycle.
+
+
+![PayNow1](images/pay-now-1.png)
+![PayNow2](images/pay-now-2.png)
+
+####Billing Cycle and Date Range 
+
+The `Pay Now` feature covers all unpaid costs up to the end of the previous billing cycle.
```
Comment (hiddenshadow21) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown:186]: Isn't it for all previous billing cycles?

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 2]
```diff
@@ -162,8 +163,47 @@ cost, and the expected charge at the end of this month.
...
+
+![Email](images/known-payment-failure-email.png)
+
+---
+
+####Pay Now 
+
+The `Pay Now` button becomes visible on your Billing tab dashboard once there are any unpaid costs from previous billing cycle(s).
+If you have resolved the issue associated with the charge failure (e.g., updated payment information), you can manually initiate the payment by clicking the `Pay Now` button.
+Upon clicking the `Pay Now` button, the system will process the payment for all outstanding amounts up to the end of the previous billing cycle.
```
Comment (PFYasu) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown:178]: In addition please add a note that this button (PayNow) is not visible in the billing day (27th day of a month)

Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown:178]: It should be wrapped into {INFO} or {DANGER / WARN} section.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown (hunk 1 above)
Comment (matz985) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown:186]: It is, it covers all unpaid costs up to the end of the previous cycle. Right under this line, there's also an example that shows how it works.

**Review by hiddenshadow21 — APPROVED** [review]

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 3]
```diff
@@ -162,8 +163,52 @@ cost, and the expected charge at the end of this month.
...
+####Pay Now 
+
+The `Pay Now` button becomes visible on your Billing tab dashboard once there are any unpaid costs from previous billing cycle(s).
+
+{INFO: }
+There is one exception to this rule, on billing day (27th of the month) `Pay Now` functionality is disabled, so the button won't be visible.
+{INFO/}
+
+If you have resolved the issue associated with the charge failure (e.g., updated payment information), you can manually initiate the payment by clicking the `Pay Now` button.
+Upon clicking the `Pay Now` button, the system will process the payment for all outstanding amounts up to the end of the previous billing cycle.
```
Comment (PFYasu) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown:183]:
```suggestion
Upon clicking the `Pay Now` button, the system will process the payment for all outstanding amounts up to the end of the previous billing cycle. If the payment process fails, you will be notified again by email.
```

**Review by PFYasu — APPROVED** [review]
Overall LGTM.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 4]
```diff
@@ -162,8 +163,52 @@ cost, and the expected charge at the end of this month.
...
+{NOTE/}
+
+####Notifications
+
+  If the payment fails, you will receive an email notification with the details and possible reasons for the failure.
+
+{INFO: }
+
+Using `Pay Now` button is only available for use once every hour.
+
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown:206]:
```suggestion
Using `Pay Now` button is only available for use once every hour.

```

I would not mention that

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-billing-tab.markdown [hunk 5]
```diff
@@ -29,6 +29,7 @@ The Billing & Costs tab will show your existing resources and their costs as wel
   The `Pay Now` button is designed to facilitate immediate payment of outstanding costs accrued from previous billing
   cycles. This feature is especially useful for ensuring timely payments and maintaining up-to-date account status.
   It can be used when automatic charge fails due to reasons like expired cards or insufficient funds.
+  You can read more about the _Pay Now_ feature on the [Pricing, Payment and Billing](../cloud-pricing-payment-billing#pay-now) page.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-billing-tab.markdown:32]:
```suggestion
  You can read more about the _Pay Now_ option on the [Pricing, Payment and Billing](../cloud-pricing-payment-billing#pay-now) page.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 6]
```diff
@@ -162,8 +163,50 @@ cost, and the expected charge at the end of this month.
 ---
 
 ####Charging failures  
-If we fail to charge your account using the payment method you provided, we'll notify 
-you and retry for 7 days. Failure to pay after that period may result in account closure. 
+If we fail to charge your account using the payment method you provided, we'll notify
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown:166]:
```suggestion
If we fail to charge your account using the payment method on record, we'll notify
```

**Review by gregolsky — APPROVED** [review]

**Pull request merged**